aemedge/blocks/zipcode/zipcode.css

```diff
@@ -146,14 +147,21 @@ header nav[aria-expanded="false"] .zipcode-wrapper {
 
 .zip-update-btn {
     margin-bottom: 20px;
-    width: 100%;
+    width: 265px;
```

Comment:
this is too wide for mobile
![image](https://github.com/user-attachments/assets/bb65d0b7-01d5-461d-bbab-6038d795b888)